infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -527,8 +527,6 @@ private synchronized void handleResetLogUnit(RequestMsg req, ChannelHandlerConte
 
         serverContext.setLogUnitEpochWaterMark(req.getPayload().getResetLogUnitRequest().getEpoch());
 
-        batchProcessor.restart();
```

Comment:
This is what's being disabled. Before PR3538, the restart of the log unit was purely done by adding a task to the BP's queue. This call to restart would reset the BP thread, but the heal node workflow was never getting this far, so this functionality wasn't being exercised.

Reply:
Hi @zfrenette, Could you please add this crucial info in the description of the PR as well? It would help to understand why these changes are being made.